repository/confluent-schema-registry/docs/v1.0/security.md

```diff
@@ -0,0 +1,103 @@
+# Security
+
+The Confluent Schema Registry service supports native transport **encryption**, **authentication**, and **authorization** mechanisms to connect with Kafka’s brokers configured for secured connections. The service provides automation and orchestration to simplify the use of these important features. For more information on Confluent Schema Registry’s security while connecting with Kafka Brokers over secured connections, read the [security](https://docs.confluent.io/current/kafka/encryption.html#sr) section of official Confluent Schema Registry documentation.
+
+## Encryption
+
+By default, KUDO Kafka brokers use the plaintext protocol for its inter-broker communication. It is recommended to enable the TLS encryption, to secure the communication between brokers. The Confluent Schema Registry can be configured to connect with Kafka brokers over a secure connection with an appropriate encryption mechanism.
+
+### Enabling TLS encryption
+
+To connect Confluent Schema Registry with Kafka Brokers configured with TLS, get the TLS certificate and key that is used for Kafka TLS encryptions to create a new Kubernetes TLS secret for Confluent Schema Registry. 
+
+Here, we are assuming that the TLS Certificate file name is `tls.cert` and key name is `tls.key` that is used for Kafka TLS encryptions. We create a new Kubernetes TLS secret for Confluent Schema Registry as:
+
+```
+$ kubectl create secret tls confluent-schema-registry-tls --cert=tls.crt --key=tls.key
+```
+
+:warning: Make sure to create the certificate in the same namespace where the KUDO Kafka is being installed.
```

Comment:
same namespace as `confluent schema registry` is being installed

Reply:
I have made correction. Thanks for the review @zmalik!